src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1525,16 +1524,15 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
             var request = new Autogenerated.GetWorkflowRequest()
             {
                 InstanceId = instanceId,
-                WorkflowComponent = workflowComponent,
-                WorkflowType = workflowName //TODO: Change 'WorkflowType' to 'WorkflowName' once changes go through dapr/dapr
+                WorkflowComponent = workflowComponent
             };
 
             try
             {
                 var options = CreateCallOptions(headers: null, cancellationToken);
                 var response = await client.GetWorkflowAlpha1Async(request, options);
-                var dateTimeValue = new DateTime(response.StartTime, DateTimeKind.Utc);
```

Comment:
Not necessarily related to this change, but wonder if `DateTimeOffset` should be used in the public APIs. (It's the generally preferred type for most application purposes.)